test/e2e/serial/tests/workload_placement_nodelabel.go

```diff
@@ -211,6 +224,193 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
+
+			By("Verifing the NRT statistics are updated")
+			targetNodeNRTCurrent, err := e2enrt.FindFromList(nrtCandidates, targetNodeName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(e2enrt.CheckEqualAvailableResources(*targetNodeNRTInitial, *targetNodeNRTCurrent)).To(BeTrue())
+		})
+
+		Context("label two nodes with different label values but both matching the node affinity of the deployment pod of the test", func() {
+			var err error
+			var unlabelTarget, unlabelAlternative func() error
+
+			BeforeEach(func() {
+				By(fmt.Sprintf("Labeling target node %q with label %q:%q and the alternative node %q with label %q:%q", targetNodeName, labelName, labelValueLarge, alternativeNodeName, labelName, labelValueMedium))
+
+				unlabelTarget, err = labelNodeWithValue(fxt.Client, labelName, labelValueLarge, targetNodeName)
+				Expect(err).NotTo(HaveOccurred(), "unable to label node %q", targetNodeName)
+
+				unlabelAlternative, err = labelNodeWithValue(fxt.Client, labelName, labelValueMedium, alternativeNodeName)
+				Expect(err).NotTo(HaveOccurred(), "unable to label node %q", alternativeNodeName)
+			})
+
+			It("[test_id:47597][tier2] should be able to schedule pod with affinity property requiredDuringSchedulingIgnoredDuringExecution on the available node with feasible numa zone", func() {
+				By("create a deployment with one guaranteed pod with node affinity property: requiredDuringSchedulingIgnoredDuringExecution")
+				deploymentName := "test-dp"
+				var replicas int32 = 1
+
+				podLabels := map[string]string{
+					"test": "test-dp",
+				}
+				nodeSelector := map[string]string{}
+				deployment := objects.NewTestDeployment(replicas, podLabels, nodeSelector, fxt.Namespace.Name, deploymentName, objects.PauseImage, []string{objects.PauseCommand}, []string{})
+				deployment.Spec.Template.Spec.SchedulerName = serialconfig.Config.SchedulerName
+				deployment.Spec.Template.Spec.Containers[0].Resources.Limits = requiredRes
+				affinity := createNodeAffinityRequiredDuringSchedulingIgnoredDuringExecution(labelName, []string{labelValueLarge, labelValueMedium}, corev1.NodeSelectorOpIn)
+				deployment.Spec.Template.Spec.Affinity = affinity
```

Comment:
nit: can we avoid the temporary value?

Reply:
changed the code a bit so we'll need the temp value to show what affinity property we are working with in each test